docs/instructors/6-codaveri/A-feedback/index.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: /instructor-guide/codaveri/feedback
+title: 6a. Automated Feedback using Codaveri
+sidebar_label: 6a. Automated Feedback
+---
+
+You may use Codaveri for automated grading feedback for coding questions.
```

Comment:
We need to first explain to them what Codaveri is. We can do it here in an admonition, or a separate index page for the Codaveri section.